docs/configuration.md

```diff
@@ -8,6 +8,7 @@ and variables needed to properly configure SC4S for your environment.
 | Variable | Values        | Description |
 |----------|---------------|-------------|
 | SC4S_USE_REVERSE_DNS | yes or no(default) | use reverse DNS to identify hosts when HOST is not valid in the syslog header |
+| SC4S_REVERSE_DNS_KEEP_FQDN | yes or no(default) | don't extract hostname from FQDN, pass the full domain name to HOST instead |
```

Comment:
| SC4S_REVERSE_DNS_KEEP_FQDN | yes or no (default) | When enable,SC4S will not extract hostname from FQDN, and instead will pass the full domain name to HOST. |